src/FakeItEasy/Creation/IMethodInterceptionTester.cs

```diff
@@ -0,0 +1,16 @@
+namespace FakeItEasy.Creation
+{
+    using System.Reflection;
+
+    internal interface IMethodInterceptionTester
+    {
+        /// <summary>
+        /// Gets a value indicating whether the specified method can be intercanepted by  instance.

```

Comment:
"intercanepted" ?

Reply:
Yes. That's not a word. I don't know how I managed that.